openapi.yaml

```diff
@@ -1512,34 +1420,35 @@ paths:
                       mva: 170 
           application/json:
             schema:
-              $ref: '#/components/schemas/recourse-rating-submission'                                          
-      responses: 
+              $ref: '#/components/schemas/recourse-rating-proposal'                                          
+      responses: &recourseProposalUpdateResponse
         "202": 
           description: >
             The update was accepted for later processing. Updates to 
             ratings may need to undergo additional validation and propagation
             to other systems.
           content:
-            application/vnd.trolie.recourse-submission.v1+json:
+            application/vnd.trolie.recourse-proposal.v1+json:
               schema:
-                $ref: '#/components/schemas/recourse-rating-submission'
+                $ref: '#/components/schemas/recourse-rating-proposal'
               example:
                 _links:
                   self:
-                    href: /rating-submissions/recourse/12345
-                id: "12345"
+                    href: /rating-proposals/recourse/UTILITYA
                 timestamp: "2023-07-12T15:05:43.044267100-07:00"
-                submitter: utilityA
+                owner: UTILITYA
                 ratings: 
                 - _links:
                     segment:
                       href: /reference-data/network-segments/segmentX
                       name: segmentX  
+                  segment-id: segmentX
                   seasons:
                   - _links:
                       season:
-                        name: fall
                         href: /reference-data/seasons/fall
+                    season-id: fall
```

Comment:
I think this is good, but I'm wondering if a season should primarily be nominated by a start-date. It could be that we support aliases, but not as the canonical identifier.

Reply:
I actually had some interesting conversations with Brian Jennings at ISO New England today about seasons.  I think we will do some significant revisiting here.  We'll have to talk about that soon.